environments/gcp/flyte-core/services.tf

```diff
@@ -0,0 +1,14 @@
+locals {
```

Comment:
Can you please check whether cloud sql and storage need to be activated here as well in case the user starts with a fresh gcp project?

Reply:
Tested again on two fresh projects and neither cloudsql or storage services need to be activated by the user